labels/de-DE.json

```diff
@@ -31,6 +31,7 @@
   "applyCredit": "Bewerben Kredit",
   "availableBalance": "Verfügbares Guthaben",
   "availableNow": "Jetzt erhältlich",
+  "backorderEst": "Geschätztes Wiederauffüllungsdatum",
```

Comment:
what's this for?

Reply:
Actually, this came from the merged commit. It is not used for this ticket

Reply:
Can you remove it if it's not used in this ticket?

Reply:
done